weedcoco/draft export to elastic.py

```diff
@@ -31,6 +31,12 @@
         id_lookup[key, obj["id"]] = obj
 
 
+for agcontext in coco["agcontexts"]:
+    # massage for ES
+    if agcontext.get("camera_fov") == "variable":
```

Comment:
Unfortunately it looks like we will need to stick with this "massaging" to make this work. Elasticsearch will not allow a field to have multiple potential types, and I think it would be a bit silly to make this into some sort of nested field array where we have something like:
```
camera_fov: {
 is_fixed: False
 fov: null
}
```

Reply:
I'm okay with this massaging, as long as any slider over FOV that we have also allows us to retrieve NULLs.